File: 2.0/python/me/findApiCredentials.py
from callfire.client import CallfireClient

client = CallfireClient('account-login', 'account-password')
response = client.me.findApiCredentials(
    # return only specific fields
    fields='name,enabled'
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/me/findApiCredentials.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("account_login", "account_password");
        $request = $client->findApiCredentials();
        $request->getOperationConfig()->setQueryParameters(array("fields" => "name,enabled"));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
